charts/aws-service-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "aws-service-operator.fullname" . }}
+  labels:
+    app: {{ template "aws-service-operator.name" . }}
+    chart: {{ template "aws-service-operator.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
These are reused in a few places, can we make a helper template for them?

Reply:
Yep, i'll also factor in the new naming standard: https://github.com/helm/helm/issues/4335